invoice-spec.json

```diff
@@ -59,6 +60,22 @@
       "items": {
         "$ref": "#/definitions/name-value"
       }
+    },
+    "name-number-value": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "name",
+        "value"
+      ],
+      "properties": {
+        "name": {
+          "$ref": "#/definitions/restricted-string"
+        },
```

Comment:
I think it can be useful to restrict metric names as well now when we're in a green-field application space.